### ISSUE TITLE
Error rendering match results

### ISSUE THREAD

**Issue by sn3p** [sn3p/utstats#17]
**Example:**
http://pgaming.site.nfoservers.com/utstats/?p=match&mid=11450

```
Fatal error: Uncaught ArgumentCountError: Too few arguments to function jsonEncodeSingleSerie(), 4 passed in /usr/www/pgaming/public/utstats/includes/renderer-general-output.php on line 144 and at least 5 expected in /usr/www/pgaming/public/utstats/includes/renderer-general-output.php:89 Stack trace: #0 /usr/www/pgaming/public/utstats/includes/renderer-general-output.php(144): jsonEncodeSingleSerie(Array, Array, 'player', 'bar') #1 /usr/www/pgaming/public/utstats/includes/renderer-general-output.php(38): jsonEncodeSeries(Array, Array, 'player', 'bar') #2 /usr/www/pgaming/public/utstats/pages/match_info_charts.php(59): renderChart('11450-1', '\n\t\ttitle : {\n\t\t...', 'player', '', Array, Array, false, 700, 250, 'bar') #3 /usr/www/pgaming/public/utstats/pages/match_info.php(14): include('/usr/www/pgamin...') #4 /usr/www/pgaming/public/utstats/pages/match.php(8): include('/usr/www/pgamin...') #5 /usr/www/pgaming/public/utstats/index.php(54): include('/usr/www/pgamin...') #6 /usr/www/pgaming/public/utstats/index.php(12): ma in /usr/www/pgaming/public/utstats/includes/renderer-general-output.php on line 89
```

**Comment by randommonk** [comment]
I think it just misses an empty string for the 5th argument, and that should solve it? These were 'warns' up until php 7.0 but php 7.1 upgraded them to errors:

http://php.net/manual/en/migration71.incompatible.php

**Comment by randommonk** [comment]
Yep makes sense, since it's what's happening already it's the safest way to have no regression

**Comment by sn3p** [comment]
Fixed in #18
